resource/vehicleProperties/client.lua

```diff
@@ -328,15 +335,17 @@ function lib.setVehicleProperties(vehicle, props)
             ClearVehicleCustomPrimaryColour(vehicle)
             SetVehicleColours(vehicle, props.color1 --[[@as number]], colorSecondary --[[@as number]])
         else
+            if type(props.paint1) == 'number' then SetVehicleModColor_1(vehicle, props.paint1, colorPrimary, pearlescentColor) end
```

Comment:
Should just be `if props.paint1`? Does pearlescentColor conflict with (the unused) p1_pearlescentColor?